ApiBundle/Controller/AreaFlexController.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace OpenOrchestra\ApiBundle\Controller;
+
+use OpenOrchestra\ModelInterface\Event\TemplateFlexEvent;
+use OpenOrchestra\BaseApiBundle\Controller\Annotation as Api;
+use OpenOrchestra\ModelInterface\TemplateFlexEvents;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration as Config;
+use Symfony\Component\HttpFoundation\Response;
+use OpenOrchestra\BaseApiBundle\Controller\BaseController;
+
+/**
+ * Class AreaFlexController
+ *
+ * @Config\Route("area-flex")
+ *
+ * @Api\Serialize
+ */
+class AreaFlexController extends BaseController
+{
+    /**
+     * @param string      $areaId
+     * @param string      $templateId
+     * @param string|null $parentId
+     *
+     * @Config\Route("/{areaId}/delete-in-template/{parentId}/{templateId}", name="open_orchestra_api_area_flex_column_delete_in_template")
+     * @Config\Route("/{areaId}/delete-in-template/{templateId}", name="open_orchestra_api_area_flex_row_delete_in_template")
```

Comment:
Maybe it could be good to invert the parameter order (parentId <=> templateId). That way we would have a standard url /{areaId}/delete-in-template/{templateId} with an optional parameter {parentId} altering the behavior of the action if present. I think the naming would be more logical that way